packages/inter-protocol/src/vaultFactory/math.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file calculations specific to the Vault Factory contract
+ * See also ../interest-math.js
+ */
+
+import { AmountMath } from '@agoric/ertp';
+import { getAmountOut } from '@agoric/zoe/src/contractSupport/priceQuote.js';
+import { floorDivideBy } from '@agoric/zoe/src/contractSupport/ratio.js';
+
+/**
+ *
```

Comment:
Please add a descriptive comment or drop the blank line.

e.g. "return the maximum debt allowed based on the price quote and the lesser of the liquidationMargin or the minimumCollateralization."